src/Hng.Web/Program.cs

```diff
@@ -21,8 +28,25 @@
     app.UseSwaggerUI();
 }
 
-app.UseHttpsRedirection();
+app.UseExceptionHandler(appBuilder =>
```

Comment:
@ubongedem78 good work so far. I just have 2 suggestions

- Create an Extension folder, then create a static class, call it `LoggerExtensions`, Create an extension method called `UseGlobalErrorHandler` and move this configuration into that extension method
- Check if the environment is development return the full exception not just the `"An unexpected fault happened. Try again later."` text.